test/findTest.js

```diff
@@ -0,0 +1,61 @@
+var db = require('../db')
```

Comment:
in unit tests, we encompass each unit in it's own file. since there is one unit - mingo. there should be one file. in that file you can have the suitable bdd split that you entended here